src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -157,7 +159,7 @@ public void teleportHome(@NotNull Player player, @NotNull Island island, int del
      */
     private void teleportHome(@NotNull Player player, @NotNull Island island) {
         player.setFallDistance(0);
-        player.teleport(LocationUtils.getSafeLocation(island.getHome(), island));
+        PaperLib.teleportAsync(player, LocationUtils.getSafeLocation(island.getHome(), island), PlayerTeleportEvent.TeleportCause.PLUGIN);
```

Comment:
we has used this in 2.x, but why not using in 3.x ?.. (dum peaches)
anyway great commit :panda: